keps/sig-release/1498-kubernetes-yearly-support-period/README.md

```diff
@@ -207,17 +207,17 @@ Unchanged from existing Kubernetes project deprecation criteria.  Deprecations f
 Incremental change elongating existing Kubernetes project version skew policy, i.e.: where “N+/-1” to “N+/-2” and where “N+/-2” to “N+/-3”.
 This matters, especially for external clients programmatically interacting with a long-lived cluster, beyond the cluster control plane components, moved through the upgrade process.
 
-## Implementation History
+Note: The above assumes no change in the number of releases made per year.  If the number of releases per year increases or decreases, that would additional modify the version skew strategy.
```

Comment:
```suggestion
Note: The above assumes no change in the number of releases made per year. If the number of releases per year increases or decreases, that may additionally modify the version skew strategy.
```

Reply:
Two spaces!!

But seriously the would versus may is an important distinction.  Fixed.